_ml-commons-plugin/model-serving-framework.md

```diff
@@ -0,0 +1,250 @@
+---
+layout: default
+title: Model serving framework 
+has_children: false
+nav_order: 110
+---
+
+# Model serving framework
+
+ML commons supports the ability to serve custom models and use those models to make inferences. For those who wish to use train their models using a deep-learning framework, such as Pytorch, they can upload such models to OpenSearch in order to take advantage of OpenSearch resources.
+
+This page outlines the steps required to upload a custom model with the ML Commons plugin.
```

Comment:
I think this page is not just explaining `upload a custom model`. 

How about change to 

```suggestion
This page outlines the steps required to upload and run a custom model with the ML Commons plugin.
```